### DISCUSSION TITLE
Request: Add GitHub Copilot icon

### DISCUSSION THREAD

**Discussion by mawkler** [ryanoasis/nerd-fonts#790] [Ideas]
[GitHub Copilot](https://copilot.github.com/) was announced last year and now it's also available for Neovim: [copilot.vim](https://github.com/github/copilot.vim). In VSCode there's an icon in the bottom right to indicate whether it is enabled for the current filetype, which I think would be very neat to be able to add to Neovim's statusline as well, for instance.

I think this is what the glyph that VSCode uses looks like:
![copilot](https://user-images.githubusercontent.com/15816726/154671367-257e8973-203d-47bd-8791-64962abc715d.png)

**Comment by twiddli** [comment]
Octoicons seems to have this available now. Not sure from what version: https://primer.style/design/foundations/icons/copilot-96
Would be nice to have these fonts updated as well.

**Comment by twiddli** [comment]
I found this: 
